scripts/docker/neo4j/init.sh

```diff
@@ -4,4 +4,4 @@
 echo "~~~"
 echo "~ Init database"
 echo "~~~"
-su-exec neo4j bin/cypher-shell -u neo4j -p admin < /source/scripts/docker/neo4j/initdb.gql
+bin/neo4j start && sleep 10 && bin/cypher-shell -u neo4j -p neo4j < /source/scripts/docker/neo4j/initdb.gql
```

Comment:
I don't understand why you have changed this line. Here the Neo4j instance is already running so you don't need to start it. Moreover the sleep `10` is a magic number :)

Reply:
When testing this on local docker I had to start neo in a couple of instances, I'll revert this.